packages/libs/client-utils/src/marmalade/index.ts

```diff
@@ -1 +1,9 @@
 export * from './create-token-id';
+
+export function hi() {
+  return 'hi';
```

Comment:
This should return `Hi!`

```suggestion
  return 'Hi!';
```